doc/advanced-topics.md

```diff
@@ -418,6 +418,27 @@ For each attribute in the `reverse` array, an expression must be defined to calc
 notification attributes. This value will be passed to the underlying protocol with the `object_id` name. Details about
 how the value is then progressed to the device are protocol-specific.
 
+#### Autoprovision configuration (autoprovision)
+
+By default, when a measure arrives to the IoTAgent, if the `device_id` does not match with an existing one, then, the IoTA 
+creates a new device and a new entity according to the group config. Defining the parameter `autoprovision` to `false` 
+when provisioning the device or the group, the IoTA to reject the measure at the southbound, allowing only to persist the 
+data to devices that are already provisioned. 
+
+#### Explicitly defined attributes (explicitAttrs)
+
+If the parameter is not defined in the device or group provision, this parameter is stored in the Context Broker by adding 
+a new attribute to the entity with the same name of the parameter. By adding the parameter `explicitAttrs` with `true` value
+to device or group provision, the IoTAgent rejects the parameters received from the measure that are not defined or in the 
+device or group provision, persisting only the one defined in the provision.
+
```

Comment:
I'd suggest to add the following paragraph:

> `If `explicitAttrs` is provided both at device and group level, the device level takes precedence

Does it work that way, right?

Reply:
Added in https://github.com/telefonicaid/iotagent-node-lib/pull/1068/commits/1c058365f8b3fd607d91def65af548f7c86a27d8